lib/io_buffer.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:io';
 /// - List<int> (memory buffer)
 /// - File (will call openRead())
 /// - String (will call utf8.encode())
-Stream<List<int>> openBytesStream(
+Stream<List<int>>? openBytesStream(
```

Comment:
I think it would be better to require a non-nullable input here.

Reply:
Done